### PR TITLE
Update 2017-01-31-early-marketplace-adoption-pays-off.md

### PR DESCRIPTION
**Pull request by skpandey27**
Define abbreviations and add a missing comma.

### CODE REVIEW

File: _posts/2017-01-31-early-marketplace-adoption-pays-off.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ Chris: The benefit for us was, even if we do have to bring in our own procuremen
 
 On screen: Anything else you would like to share about your experience?
 
-Chris: The simplicity of the language was a real strong point for my experience of the Marketplace - not having the RFQs the ROIs and the tenders. I'm just someone who needs a solution to my problem, and anyone who can help me connect with people who provide that solution I'm really happy with, and if it's in plain English, all the better.
+Chris: The simplicity of the language was a real strong point for my experience of the Marketplace - not having the Request for Quotations (RFQs), the Return on Investments (ROIs) and the tenders. I'm just someone who needs a solution to my problem, and anyone who can help me connect with people who provide that solution I'm really happy with, and if it's in plain English, all the better.
```
Comment (bjgordon) [inline _posts/2017-01-31-early-marketplace-adoption-pays-off.md]: We can't rewrite the transcript to what he didn't say. Please change to use the [`<abbr>` tag](http://www.w3schools.com/tags/tag_abbr.asp) like we did on the ab coding post.

Comment (skpandey27) [inline _posts/2017-01-31-early-marketplace-adoption-pays-off.md]: Modified with tag.

**Comment by bjgordon** [comment]
Thanks @skpandey27
